packages/devtools-reps/src/reps/test-helpers.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think you could try to exclude the helper file by appending its path to `testPathIgnorePattern` (see https://facebook.github.io/jest/docs/configuration.html#testpathignorepatterns-array-string) in https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-reps/package.json#L49